tests/expected_cmdline_output_files/basic_connectivity_expl_output.txt

```diff
@@ -0,0 +1,14775 @@
+final fw rules for query: , config: testcase8-networkpolicy1.yaml:
+src: 0.0.0.0/0 dst_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] dst_pods: [*] conn: All connections
+src_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
+src_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] src_pods: [*] dst_ns: [default,kube-system,kube-system-dummy-to-ignore,vendor-system] dst_pods: [*] conn: All connections
+<entry dst="0.0.0.0/0" id="1" src="0.0.0.0/0">
+ <text>
+  Configurations affecting 0.0.0.0/0:
+IP blocks have no configurations
+ </text>
```

Comment:
I think we can remove this entire section. it does not add any information.

Reply:
Done

Reply:
I still see this section:
```
 <text>
  Configurations affecting 0.0.0.0/0:
 </text>
```

Reply:
no need to create entry for `<entry dst="0.0.0.0/0" id="1" src="0.0.0.0/0">` , is both src and dst are ip- blocks it should be skipped.